.github/workflows/go.yaml

```diff
@@ -51,12 +23,17 @@ jobs:
     - name: Run go mod tidy on all modules
       run: find . -name go.mod -execdir go mod tidy \;
     # If there are any diffs from goimports or go mod tidy, fail.
-    - name: Verify no changes from goimports and go mod tidy. If you're reading this and the check has failed, run `goimports -w . && go mod tidy` or add the `autoformat` label to the PR.
+    - name: Verify no changes from goimports and go mod tidy. If you're reading this and the check has failed, run `goimports -w . && go mod tidy`.
       run: |
         if [ -n "$(git status --porcelain)" ]; then
           git status # Show the files that failed to pass the check.
           exit 1
         fi
+    - name: go vet
+      # Use find to build all modules. '-execdir ... ;' doesn't set an exit code
+      # based on command results. So, create a file if a build fails and check
+      # if the file exists to set the right exit code.
+      run: find . -name go.mod -execdir sh -c 'go vet ./... || touch vet_failed.txt' \; ; test ! -f vet_failed.txt
```

Comment:
consider splitting it out to a shell script in the testing directory if it's going to get complicated

Reply:
I think that will be the way to go in the future. I'd like to keep the automatic parallelism, but this is a gnarly line of Bash.